data/org.freedesktop.portal.ScreenCast.xml

```diff
@@ -169,6 +200,14 @@
               'multiple' was set to 'true' as part of the SelectSources method.
             </para></listitem>
           </varlistentry>
+          <varlistentry>
+            <term>restore_token s</term>
+            <listitem><para>
+              The restore token.
+
```

Comment:
This should probably mention that the token in a single-use token, and it will only be allowed to be used to restore a session once. Maybe e.g.

```
This token is a single use token that can later be used to restore
a session. See  org.freedesktop.portal.ScreenCast.SelectSources
for details.
```

Reply:
Added too